packages/react-components/src/organisms/__tests__/WorkingGroupCard.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { ComponentProps } from 'react';
+import { render } from '@testing-library/react';
+
+import WorkingGroupCard from '../WorkingGroupCard';
+
+const props: ComponentProps<typeof WorkingGroupCard> = {
+  id: '42',
+  title: 'My Working Group',
+  externalLink: 'https://www.google.com',
+  externalLinkText: 'Working Group Folder',
+  description: 'My Working Group Description',
+  lastModifiedDate: '2020-01-1',
+};
+
+it('renders the working group card', () => {
+  const { getByText } = render(
+    <WorkingGroupCard
+      {...props}
+      id="42"
+      title="My Working Group"
+      description="test description"
+      externalLinkText="Link text"
+      lastModifiedDate="2020-01-01"
+    />,
+  );
+  expect(getByText('My Working Group')).toBeVisible();
+  expect(getByText('test description')).toBeVisible();
+  expect(getByText('Link text')).toBeVisible();
+  expect(getByText('Last updated: 2020-01-01')).toBeVisible();
```

Comment:
I think it should be something like this

```
expect(getByText('Last updated: 1st January 2022')).toBeVisible();
```

instead of `2020-01-01` right? following the pattern of other cards like this one
<img width="752" alt="Screenshot 2022-11-29 at 11 29 36" src="https://user-images.githubusercontent.com/16595804/204556236-d9065e1e-59e0-4ed2-88d0-0257183a2812.png">